.golangci.yml

```
@@ -0,0 +1,79 @@
+linters:
+  enable:
+  - asciicheck
+  - dogsled
+  - durationcheck
+  - exhaustive
+  - exportloopref
+  - gci
+  - goconst
+  - gofmt
+  - gofumpt
+  - goimports
+  - gomodguard
+  - gosec
+  - govet
+  - importas
+  - lll
+  - megacheck
+  - misspell
+  - nakedret
+  - nilerr
+  - nolintlint
+  - predeclared
+  - revive
+  - stylecheck
+  - unconvert
+  - unparam
+  - wastedassign
+  - errorlint
+issues:
```

```suggestion
issues:
  max-same-issues: 0
```

This will make us immediately get the full occurrence list of (the same) issues instead of capping it to 3.